frontend2/src/components/tables/chart/TeamChart.tsx

```diff
@@ -17,18 +17,23 @@ export type ChartData = [UTCMilliTimestamp, number];
 // the second is the ranking of that team. Example:
 //
 // { "Gone Sharkin" : [ [ 1673826806000.0, 1000 ], [1673826805999.0, 900], ... ], ...}
+//
+// `plotLines` holds the tournament lines. You use them like this:
+// [ ["Tournament 1", 122342982341], ["Tournament 2", 122345081100], ...]
 export interface TeamChartProps {
   yAxisLabel: string;
   values?: Record<string, ChartData[]>;
   loading?: boolean;
   loadingMessage?: string;
+  plotLines?: Array<[string, UTCMilliTimestamp]>;
```

Comment:
nit: would a name like `verticalLabels` or similar be more descriptive of what these actually look like/are?